Source/WebCore/rendering/PositionedLayoutConstraints.cpp

```diff
@@ -353,7 +353,10 @@ LayoutUnit PositionedLayoutConstraints::resolveAlignmentShift(LayoutUnit unusedS
 
     LayoutUnit shift;
     if (ItemPosition::AnchorCenter == resolvedAlignment) {
-        auto anchorCenterPosition = m_anchorArea.min() + (m_anchorArea.size() - itemSize) / 2;
+        auto anchorCenterPosition = LayoutUnit();
+        anchorCenterPosition = m_anchorArea.min() + (m_anchorArea.size() - itemSize) / 2;
```

Comment:
I wonder if it would be possible to keep this in one line, by changing the `auto` type to use `LayoutUnit` explicitly. Maybe this would work:

```suggestion
        LayoutUnit anchorCenterPosition = m_anchorArea.min() + (m_anchorArea.size() - itemSize) / 2;
```